index.html

```diff
@@ -0,0 +1,371 @@
+							<a class="navbar-brand text-dark" href="index.html"
+								><img
+									class="victoria-logo"
+									src="./asset/images/victoria-university-logo.png"
+									alt="Victoria-university-logo" />
+							</a>
+
+							<ul
+								class="navbar-nav conatiner-fluid d-flex justify-content-between align-items-center">
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a class="nav-link text-dark" href="./index.html">Home</a>
+								</li>
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a
+										class="nav-link text-dark"
+										id="desktop-nav"
+										href="./about.html"
+										>About</a
+									>
+								</li>
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a class="nav-link text-dark" href="#">Program</a>
+								</li>
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a class="nav-link text-dark" href="#">Sponsor</a>
+								</li>
+								<li class="nav-item mx-4" id="desktop-nav">
+									<a class="nav-link text-dark" href="#">Log out</a>
+								</li>
+								<li class="nav-item mx-4 border border-5 border-danger">
+									<a class="nav-link text-center text-danger" href="#"
+										>Our legacy</a
+									>
+								</li>
+							</ul>
+						</div>
+					</nav>
+				</div>
+			</header>
+
+			<main>
+				<!--Intro section -->
+				<section class="hero-container">
+					<div class="hero-content py-5 px-3">
+						<!-- <div class="hero-title"> -->
+						<h1 class="h1 text-danger mt-5 mb-4">
+							VICTORIA UNIVERSITY ALUMNI STUDENT SUMMIT 2023
+						</h1>
+
+						<p class="textColor border border-5 border-white p-2">
+							The university of victoria alumni Summit annually brings together
+							students, faculty, alumni and professionals to hear from
+							world-renowned industry leaders on pressing topics and trends that
+							are shaping the future of the business and will be conduct on a.
+						</p>
+						<div class="hero-event-date">
+							<h2 class="textColor mt-5">2023.08.26(MON) ~ WEN(28)</h2>
+							<p class="textColor">
+								@Victoria university,Kampala uganda main hall
+							</p>
+						</div>
+					</div>
+				</section>
+				<!-- main-program section -->
+				<section class="program-container bgColor py-5">
+					
+						<h3 class="display-4 text-light text-center">Main Program</h3>
+						<div class="d-flex justify-content-center">
+							<hr class="bar"> </hr>
+						</div>
+					
+					<!-- program of the day -->
+					<div class="program-content container">
+						<div
+							class="d-flex justify-content-between align-content-center shadow p-2 mb-2 bgColor3"
+							id="program-info">
+							<div class="program-img">
+								<img
+									class="text-center"
+									src="./asset/images/program_lecture.png"
+									alt="lecture" />
+							</div>
+
+							<h3
+								class="text-danger text-center display-5 px-2"
+								id="program-task">
+								LECTURE
```

Comment:
[REQUIRED] Kindly reduce the size of this text so that it can match the template. It is a requirement for your design to match the template in terms of layout, spacing, colour and typography.